recipes/libbluetooth-headers/meta.yaml

```
@@ -0,0 +1,28 @@
+{% set name = "libbluetooth-headers" %}
```

```suggestion
{% set name = "bluez-headers" %}
```
This package is called Bluez and every header you are installing has the Bluez name in a comment at the top. Shouldn't this package be called bluez-headers?

Why are you installing the headers into include/bluetooth instead of include/bluez?